src/test/java/net/sf/jsqlparser/util/deparser/ExecuteDeParserTest.java

```diff
@@ -18,63 +18,61 @@
 
 import org.junit.Before;
 import org.junit.Test;
-import org.mockito.Mock;
-
 import net.sf.jsqlparser.expression.Expression;
-import net.sf.jsqlparser.expression.ExpressionVisitor;
+import net.sf.jsqlparser.expression.JdbcParameter;
 import net.sf.jsqlparser.expression.operators.relational.ExpressionList;
 import net.sf.jsqlparser.statement.execute.Execute;
+import net.sf.jsqlparser.statement.execute.Execute.EXEC_TYPE;
 
 public class ExecuteDeParserTest {
-    private ExecuteDeParser executeDeParser;
 
-    @Mock
-    private ExpressionVisitor expressionVisitor;
+    private ExecuteDeParser executeDeParser;
+    private ExpressionDeParser expressionVisitor;
 
     private StringBuilder buffer;
 
     @Before
     public void setUp() {
         buffer = new StringBuilder();
+        expressionVisitor = new ExpressionDeParser();
+        expressionVisitor.setBuffer(buffer);
         executeDeParser = new ExecuteDeParser(expressionVisitor, buffer);
     }
 
     @Test
     public void shouldDeParseExecute() {
         Execute execute = new Execute();
         String name = "name";
-        ExpressionList exprList = new ExpressionList();
-        List<Expression> expressions = new ArrayList<Expression>();
-        Expression expression1 = mock(Expression.class);
-        Expression expression2 = mock(Expression.class);
 
-        execute.setName(name);
-        execute.setExprList(exprList);
-        exprList.setExpressions(expressions);
-        expressions.add(expression1);
-        expressions.add(expression2);
+        List<Expression> expressions = new ArrayList<>();
+        expressions.add(new JdbcParameter());
+        expressions.add(new JdbcParameter());
+
+        execute.withName(name)
+        .withExecType(EXEC_TYPE.EXEC).withParenthesis(true)
+        .withExprList(new ExpressionList().withExpressions(expressions));
 
         executeDeParser.deParse(execute);
 
         String actual = buffer.toString();
-        assertTrue(actual.matches("EXECUTE " + name + " .*?, .*"));
+        assertEquals("EXEC " + name + " (?, ?)", actual);
```

Comment:
Here the outcome is suddenly EXEC not EXECUTE. Why?

Reply:
reverted it to be EXEC - see https://github.com/JSQLParser/JSqlParser/pull/1014#discussion_r454763565 too